commands/operator-sdk/cmd/up/local.go

```diff
@@ -18,20 +19,22 @@ func NewLocalCmd() *cobra.Command {
 	upLocalCmd := &cobra.Command{
 		Use:   "local",
 		Short: "Launches the operator locally",
-		Long: `The operator-sdk up local command launches the operator on the local machine 
-by building the operator binary with the ability to access a 
+		Long: `The operator-sdk up local command launches the operator on the local machine
+by building the operator binary with the ability to access a
 kubernetes cluster using a kubeconfig file.
 `,
 		Run: upLocalFunc,
 	}
 
 	upLocalCmd.Flags().StringVar(&kubeConfig, "kubeconfig", "", "The file path to kubernetes configuration file; defaults to $HOME/.kube/config")
+	upLocalCmd.Flags().StringVar(&passThroughFlags, "extra-flags", "", "The extra flags that your operator will need. Example: \"--namespace test\"")
```

Comment:
I think we might want to make `"extra-flags"` more explicit such as `"operator-flags"`

I think change the description to:
`"The flags that the operator needs. Example: \"--flag1=value1 --flag2=value2 ...\""`
sounds a bit more formal to avoid `your` and `will`.